app/serializers/keyword_detail_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class KeywordDetailSerializer < ApplicationSerializer
+  attributes :name
+  attributes :ads_top_count
+  attributes :ads_page_count
+  attributes :ads_top_urls
+  attributes :result_count
+  attributes :result_urls
+  attributes :total_link_count
+  attributes :html
+  has_one :source
```

Comment:
```suggestion

  has_one :source
```

Optional but it might be interesting to add an empty line in this case 💭